pkg/manager/manager_test.go

```diff
@@ -845,11 +850,13 @@ var _ = Describe("manger.Manager", func() {
 		m, err := New(cfg, Options{})
 		Expect(err).NotTo(HaveOccurred())
 		startGoruntime := rt.NumGoroutine()
+		_ = pprof.Lookup("goroutine").WriteTo(os.Stdout, 1)
```

Comment:
The test here is apparently flaky with `-race` set. I am not sure if we can fix this, but having the dump here will at least give some clues.

I am unfortunately not able to repro the issue on my computer, that's why I left this here.